.gitignore

```diff
@@ -1,61 +1,62 @@
-settings_local.py
-settings/local.py
-*/settings/local.py
 __pycache__/
-*.py[cod]
-*$py.class
-*.sw[po]
+!.vscode/extensions.json
+!root_files/bedrock_db_info.json
+.#*
+.cache
 .coverage
-.jshintrc
-pip-log.txt
-docs/_gh-pages
-docs/_build
-build.py
-build
+.docker-build*
 .DS_Store
-*-min.css
+.env
+.eslintcache
+.idea
+.jshintrc
+.noseids
+.python-version
```

Comment:
This file was so long, I sorted alphabetically. But this line is the one addition here.